src/header/navigation.rs

```diff
@@ -81,16 +81,11 @@ pub fn Header() -> impl IntoView {
             ..Default::default()
         },
         NavEntry {
-            href: "#".to_string(),
-            text: "More".to_string(),
-            icon: NavIcon::More,
+            href: "/broadcast".to_string(),
```

Comment:
```suggestion
            href: "/send".to_string(),
```

Seems like we might want to change all the terminology to `send` (including in URLs).

Reply:
Decided against this.